### PR TITLE
Update ed.json

### PR DESCRIPTION
**Pull request by randyhart**

### CODE REVIEW

File: ed.json [hunk 1]
```diff
@@ -38,4 +38,11 @@
   "I mean, if I’m honest, it’s just to stick it to Randy and all those other cocky-Fredericksburgians. I’m glad their art mart closed. they don’t deserve art.",
   "Randy and I are going to give a speech  about using Bottle Rocket as a guide to digital transformation.",
   "@mikegintz you’re in the god forsaken Fredricksburg channel because I thought you might like the content of this thread."
+  "I think the way you could make it thread compliant is to say something like, 'What is such an adorable cat as Pete doing in a dump like Fredericksburg?'",
```
Comment (tbaxter-18f) [inline ed.json]: Randy, you need a comma at the end of each item except the last. If you put one in at the end of the one before your additions, you should be fine.

File: ed.json [hunk 2]
```diff
@@ -38,4 +38,11 @@
...
+  "But it wouldn’t be _the same_ thread. I don’t want to be selfish.",
+  "I love threads",
+  "Anyone else ever come on over here to catch a little bant, see something, comment (thread of course), and then realize the bant you’re reacting to is 350 messages back/36 minutes old and get embarrassed you’re so behind?",
+  "One of the most important things I want newcomers to the thread to know is that this is your thread. Regardless of what's come earlier in the thread, you are here because you belong here. Take the thread where you want it to go. Make it reflect your hopes and dreams for the thread. Add whatever Huey Lewis and the News videos to the *random huey* response that you want. Or even make new thread-specific responses. This thread belongs to you.",
+  "But whatever you do, Randy can't know about this thread, because he ruined the last one. And that just. Can. Not. Be. Forgiven.",
+  "There’s no threading culture in this channel. IMHO, `threaded bants` would be improve bant navigation.",
```
Comment (tbaxter-18f) [inline ed.json]:
```suggestion
  "There’s no threading culture in this channel. IMHO, `threaded bants` would improve bant navigation.",
```

File: ed.json (hunk 1 above)
Comment (randyhart) [reply ed.json]: ok....taking a shot at fixing.

File: ed.json [hunk 3]
```diff
@@ -37,5 +37,12 @@
...
+  "I think the way you could make it thread compliant is to say something like, 'What is such an adorable cat as Pete doing in a dump like Fredericksburg?'",
+  "But it wouldn’t be _the same_ thread. I don’t want to be selfish.",
+  "I love threads",
+  "Anyone else ever come on over here to catch a little bant, see something, comment (thread of course), and then realize the bant you’re reacting to is 350 messages back/36 minutes old and get embarrassed you’re so behind?",
+  "One of the most important things I want newcomers to the thread to know is that this is your thread. Regardless of what's come earlier in the thread, you are here because you belong here. Take the thread where you want it to go. Make it reflect your hopes and dreams for the thread. Add whatever Huey Lewis and the News videos to the *random huey* response that you want. Or even make new thread-specific responses. This thread belongs to you.",
+  "But whatever you do, Randy can't know about this thread, because he ruined the last one. And that just. Can. Not. Be. Forgiven.",
+  "There’s no threading culture in this channel. IMHO, `threaded bants` would improve bant navigation.",
```
Comment (tbaxter-18f) [inline ed.json]:
```suggestion
  "There’s no threading culture in this channel. IMHO, `threaded bants` would improve bant navigation."
```

**Pull request merged**